schedule/yast/lvm_thin_provisioning_dev.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: lvm_thin_provisioning_dev
+vars:
+  DUD_ADDONS: sdk
+  DESKTOP: gnome
+  SCC_ADDONS: base,serverapp,desktop
+  YUI_REST_API: 1
```

Comment:
`DESKTOP` is not needed and `SCC_ADDONS` neither, because we are fixing the registration with `register_module_desktop`